### ISSUE TITLE
extension upgrade path 

### ISSUE THREAD

**Issue by rafalxs** [apache/datasketches-postgresql#59]
Hi,

In my database I have data sketches in version 1.3.0 and I would like to upgrade to 1.5.0. 

Using standard upgrade path fails: 
```
ds_upgrade_test=# alter extension datasketches update;
ERROR:  extension "datasketches" has no update path from version "1.3.0" to version "1.5.0"
```

Is there a way to create / build extension upgrade script (`datasketches--1.3.0--1.5.0.sql` ?) that would support such an update? 

I cannot go with 
```
drop extension datasketches;
create extension datasketches;
```
as large number of table's columns depend on `hll_sketch` type.

**Comment by AlexanderSaydakov** [comment]
I am not quite sure how to do this. We did not have resources so far to figure out all details.
Do you care about the hll_sketch only? If so, I don't think there are any SQL changes for that sketch. So making and installing a new .so should be sufficient. But I am not sure how to tell the database to reload the .so. And it will still think it is running 1.3.0

**Comment by jmalkin** [comment]
We are trying to look into this, but don't have a target date at this point.

Just curious: Is there any specific new feature you need from 1.5?

**Comment by rafalxs** [comment]
We don't have any specific requirement at the moment. We'd like:
- be compliant with internal patching policy (1.3 is quite old now) 
- keep our options open for future use 

Thanks for help and a patch!

**Comment by jmalkin** [comment]
@rafalxs This was also included (and extended to 1.6) in the newest official release. No need to grab an unreleased version branch directly from github!

Release and notes: [https://github.com/apache/datasketches-postgresql/releases/tag/1.6.0](https://github.com/apache/datasketches-postgresql/releases/tag/1.6.0)
